app/scripts/controllers/preferences.js

```diff
@@ -15,6 +15,7 @@ const mainNetworks = {
 const testNetworks = {
   [CHAIN_IDS.GOERLI]: true,
   [CHAIN_IDS.SEPOLIA]: true,
+  [CHAIN_IDS.LINEA_GOERLI]: true,
```

Comment:
I think we still need to get rid of goerli.

Reply:
yes, this revert PR is only temporary, and it is to make sure the deprecation only happens in v11.17